operator/src/main/java/org/keycloak/operator/crds/v2alpha1/deployment/spec/Truststore.java

```diff
@@ -21,12 +21,13 @@
 import io.sundr.builder.annotations.Buildable;
 
 import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonPropertyDescription;
 
 @JsonInclude(JsonInclude.Include.NON_NULL)
 @Buildable(editableEnabled = false, builderPackage = "io.fabric8.kubernetes.api.builder")
 public class Truststore {
 
-    @Required
+    @JsonPropertyDescription("Not used")
```

Comment:
@shawkins It would be good to add a little more text with an explanation of why it is not used, right? Or make it somehow deprecated by mentioning that it's gonna be removed in later versions?

Reply:
Added the text about removal.